Feat:Working with API

Stevenkwizera06 opened the pull request:
Rendering json data
Flipping the card
Fetch the data from the URL

Stevenkwizera06 committed:
Feat:Working with API
Rendering json data
Flipping the card
Fetch the data from the URL

NNesta commented on src/components/CardComponent.jsx:
```diff
@@ -0,0 +1,18 @@
+const CardComponent = ({lname, fname, employment, image}) => {
+    return ( 
```
use descriptive naming for lname and fname

NNesta commented on src/App.jsx:
```diff
@@ -1,3 +1,70 @@
+  };
+
+  const array = genLinks();
+
```
use descriptive naming for genLinks, array, arr and i variables

NNesta commented on src/App.jsx:
```diff
@@ -1,3 +1,70 @@
+        setUsersData(newArray);
+      } catch (error) {
+        console.error(error);
+      }
```
You can remove this console.error(error) instead stores the error in its state as it may be a good way of processing an error instead of logging it on the console.

NNesta commented on src/App.jsx:
```diff
@@ -1,3 +1,70 @@
+          lname,
+          employment,
+        }));
+        setUsersData(newArray);
```
use descriptive naming for lname, fname and newArray variables

NNesta reviewed (changes requested):
Good work but you should work on those changes

Stevenkwizera06 committed:
added the feedbacks given like
using using descriptive names
 removing  the  console.error(error) and store the error in its state as it may be a good way of processing an error instead of logging it on the console

Stevenkwizera06 committed:
corrected the suggested feedback i was given on descriptive names

NNesta reviewed (approved):
Good job
